modules/cudl-data-processing/locals.tf

```diff
@@ -9,12 +12,20 @@ locals {
       filter_prefix = notification.filter_prefix
       filter_suffix = notification.filter_suffix
       queue_names   = [for subscription in notification.subscriptions : subscription.queue_name]
-    } if contains(var.source-bucket-names, notification.bucket_name)
+    } if contains(local.transform-lambda-bucket-names, notification.bucket_name)
   }
   source_bucket_sqs_notifications = {
```

Comment:
This local variable could be renamed as it can now contain the dest bucket as well as the source bucket